website/content/docs/operators/deployment/schedulers/mesos-local-mac.md

```diff
@@ -0,0 +1,130 @@
+[2016-07-25 22:04:41 -0700] com.twitter.heron.scheduler.SubmitterMain FINE:  Topology ExclamationTopology submitted successfully
+INFO: Topology 'ExclamationTopology' launched successfully
+INFO: Elapsed time: 4.114s.
+``` 
+
+Note that this doesn't necessarily mean it's successfully launched in Mesos, to check that you can go to the working directory as shown in the output. You will see:
+
+* `heron-examples.jar`: the jar which contains the topology submitted.
+* `heron-conf`: configurations used to launch the topology.
+* `log-files`: directory containing Mesos scheduler's log.
+
+You can look at the log file to see whether the launch succeeded. If it succeeded, at the end of the log file you'll see:
+
+```bash
+[2016-07-25 22:15:47 -0700] com.twitter.heron.scheduler.mesos.framework.MesosFramework INFO:  Received status update [...]
+[2016-07-25 22:15:47 -0700] com.twitter.heron.scheduler.mesos.framework.MesosFramework INFO:  Task with id 'container_1_1469510147073:0' RUNNING
+``` 
+
+If the launch fails, you will see the error message. E.g. if the Heron can't find Mesos library in the given path, it will complain:
+
+```bash
+[2016-07-25 22:04:42 -0700] stderr STDERR:  Failed to load native Mesos library from /usr/lib/mesos/0.28.1/lib
+[2016-07-25 22:04:42 -0700] stderr STDERR:  Exception in thread "main"
+[2016-07-25 22:04:42 -0700] stderr STDERR:  java.lang.UnsatisfiedLinkError: no mesos in java.library.path
+[2016-07-25 22:04:42 -0700] stderr STDERR:      at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867)
+...
+```
+
+## Mesos Management Console
+
+Another way to check your topology is running is to look at the Mesos management console, if the topology was launched successfully you should see two containers of the topology running:
+
+![result page](/img/mesos-management-console-with-topology.png)
+
+They are containers of topology master and heron container. If you click the `sandbox` on the right side, you can access the logs of these processes. E.g. the sandbox of heron container looks as below:
+
+![container-container-sandbox](/img/container-container-sandbox.png)
+
+If you go to `log-files` directory, you will see application and GC log of the processes run in this container:
+
+![container-log-files](/img/container-log-files.png)
+
+If you click `container_1_exclaim1_1.log.0`, you will see the bolt log of the ExclamationTopology:
+
+![bolt-log](/img/bolt-log.png)
+
+## Heron UI
+
+If you have Heron tools installed (see [Quick Start Guide](../../../../getting-started)), you can also monitor topology through `heron-ui`. First you need to start the `heron-tracker`. To start the tracker, modify the value of `statemgrs.rootpath` in `$HOME/.herontools/conf/heron_tracker.yaml` to `$HOME/.herondata/repository/state/mesos`. This tells Heron tracker to go to `mesos` directory (default is `local`) to find topology state information. Now start `heron-tracker`:
```

Comment:
Install Heron tools to monitor the topology with the heron-ui, which required the heron-tracker. Configure ... before starting tracker. This configuration sets the location of the state manager root path.